0.preprocessing_data/scripts/0.patient_specific_preprocessing.py

```diff
@@ -36,13 +36,13 @@
     # comment out depending on whose computer you are on
     # mike's computer
     raw_image_dir_local = pathlib.Path(
-        "/home/lippincm/Desktop/20TB_A/NF1_Patient_organoids/"
+        "/home/lippincm/mnt/bandicoot/NF1_organoid_data/Raw_patient_files"
     ).resolve(strict=True)
     # Jenna's computer
     # raw_image_dir_local = pathlib.Path("/media/18tbdrive/GFF_organoid_data/Cell Painting-NF0014 Thawed3-Pilot Drug Screening")
 
```

Comment:
```suggestion
    # Jenna's computer
    # raw_image_dir_local = pathlib.Path("/home/jenna/mnt/bandicoot/NF1_organoid_data/Raw_patient_files")
```
I think this is the same for me so recommend changing it now instead of me changing later, if possible please :)